### PR TITLE
New package: xdgmenumaker-1.5

### PR DESCRIPTION
**Pull request by stenstorp**
<strike>txt2tags template based off Gentoo ebuild.</strike>

### CODE REVIEW

File: srcpkgs/txt2tags/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'txt2tags'
+pkgname=txt2tags
+version=2.6
+revision=1
+archs=noarch
+depends="python"
+short_desc="Converts text to various formats"
+maintainer="Thomas Batten <stenstorpmc@gmail.com>"
+license="GPL-2.0"
```
Comment (Johnnynator) [inline srcpkgs/txt2tags/template]: License should be a SPDX-Identifier, so either `GPL-2.0-or-later` or `GPL-2.0-only`

File: srcpkgs/txt2tags/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'txt2tags'
+pkgname=txt2tags
+version=2.6
+revision=1
+archs=noarch
+depends="python"
```
Comment (Johnnynator) [inline srcpkgs/txt2tags/template]: We do not accept any new python2 only packages.

File: srcpkgs/xdgmenumaker/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'xdgmenumaker'
+pkgname=xdgmenumaker
+version=1.5
+revision=1
+archs=noarch
+build_style=gnu-makefile
+makedepends="txt2tags"
+depends="python python-xdg pygtk"
```
Comment (Johnnynator) [inline srcpkgs/xdgmenumaker/template]: Same here.

**Comment by Johnnynator** [comment]
The commit messages should follow the scheme `New Package: ${pkgname}-${version}`

**Comment by abenson** [comment]
`xdgmenumaker` has support for python 3.x; I didn't check txt2tags

**Comment by stenstorp** [comment]
I can build xdgmenumaker without txt2tags, it just doesn't build the man page. Would it be better to include the manpage in the srcpkg as an extra file or not have it at all?

**Comment by Chocimier** [comment]
Another option is to convert during build with `pandoc`, manpage is not as pretty but does not need to generated outdoors then.

**Comment by stenstorp** [comment]
Just tried generating the man with pandoc. Not just ugly, absolutely unusable. 
Using the intended tool is definitely the best option.

**Comment by Chocimier** [comment]
Already added.

**Pull request closed without merging**